app/models/metric/ci_mixin.rb

```diff
@@ -1,5 +1,6 @@
 module Metric::CiMixin
   extend ActiveSupport::Concern
+  include SupportsFeatureMixin
```

Comment:
unless, every module that this mixin includes already has the SupportsFeatureMixin